delivery_carrier_option/models/delivery_carrier_template_option.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2012 Akretion <http://www.akretion.com>.
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliveryCarrierTemplateOption(models.Model):
+    """Available options for a carrier (partner)"""
+
+    _name = "delivery.carrier.template.option"
+    _description = "Delivery carrier template option"
+
+    partner_id = fields.Many2one(comodel_name="res.partner", string="Partner Carrier")
```

Comment:
```suggestion
    type = fields.Selection(
        selection=[
            ("basic", "Basic Service"),
            ("additional", "Additional Service"),
            ("delivery", "Delivery Instructions"),
            ("partner_option", "Partner Option"),
        ],
        default="basic",
        string="Option type"
    )
    partner_id = fields.Many2one(comodel_name="res.partner", string="Partner Carrier")
```

We could directly include this field from postlogistics (partial elements). Because this information is generic enough to put it there IMHO.

Reply:
Hello @StephaneMangin Thanks for the suggestion.
The only purpose is informative ? There is no additional logic about this field ?
I don't really need it, but I have nothing against I guess

Reply:
It's enough generalistic to be implemented here. And indeed, it is just informative at this stage. (but not on the delivery_postlogistics module anyway)